src/components/reusable/loaders/skeleton.scss

```diff
@@ -5,9 +5,9 @@
   --skeleton-base-width: 100%;
   --skeleton-base-height: 16px;
   --skeleton-border-radius: 4px;
-  --skeleton-background: var(--kd-color-background-ui-soft);
-  --skeleton-background-dark: var(--kd-color-background-ui-subtle);
-  --skeleton-shimmer-duration: 1.5s;
+  --skeleton-background: var(--kd-color-background-container-soft);
+  --skeleton-background-dark: var(--kd-color-background-container-subtle);
```

Comment:
Any particular reason to assign these variables to another variable?

Reply:
--skeleton-background-dark property is used in data table global filter skeleton